gutenblocks.php

```diff
@@ -104,6 +104,10 @@ private function inc() {
  * @since 1.0.0
  */
 function gutenblocks() {
+	if ( ! defined( 'GUTENBERG_VERSION' ) ) {
+		return false;
+	}
+	
```

Comment:
I've been thinking about it and i'd prefer a function check, as the `GUTENBERG_VERSION` constant is only defined into the WordPress.org plugin's repo and not into the GitHub repository (which is the one I'm using)

See https://github.com/WordPress/gutenberg/blob/master/bin/generate-gutenberg-php.php#L22